css-structure/css-structure_custom-properties/css/styles.css

```diff
@@ -4,32 +4,40 @@
   box-sizing: border-box;
 }
 
+:root {
+  --color-light-black: #303030;
```

Comment:
I know that black & white occur the same amount of times but maybe it would still be better to rename them in a way that is not the color? I think Sven talked about how if "the client" wants to change let's say "color-light-black" into yellow not, your code would say light black everywhere where it would then be yellow. Get the idea?